win32ss/user/winsrv/usersrv/init.c

```diff
@@ -121,8 +121,32 @@ CreateSystemThreads(PVOID pParam)
 
 CSR_API(SrvCreateSystemThreads)
 {
-    DPRINT1("%s not yet implemented\n", __FUNCTION__);
-    return STATUS_NOT_IMPLEMENTED;
+    HANDLE ServerThread;
+    CLIENT_ID ClientId;
+    NTSTATUS Status;
+
+    Status = RtlCreateUserThread(NtCurrentProcess(),
+                                 NULL,
+                                 TRUE,
+                                 0,
+                                 0,
+                                 0,
+                                 CreateSystemThreads,
+                                 (PVOID)TRUE,
+                                 &ServerThread,
+                                 &ClientId);
+
+    if (NT_SUCCESS(Status))
+    {
+        NtResumeThread(ServerThread, NULL);
+        NtClose(ServerThread);
+    }
+    else
+    {
+        DPRINT1("Cannot start system thread!\n");
+    }
+
```

Comment:
In case of thread creation failure, it may be nice to look at `NtUserCallOneParam()` with `ONEPARAM_ROUTINE_HANDLESYSTHRDCREATFAIL` parameter (see https://reactos.org/wiki/Techwiki:Win32k/apfnSimpleCall ).

Reply:
I know that this function exists but I don't know what its parameter should be or what it should do. This is the reason I didn't call it here.